### DISCUSSION TITLE
What sushi do you like?

### DISCUSSION THREAD

**Discussion by touri-n** [NAIST-SE/AttractingContributors#261] [General]
https://github.com/touri-n/AttractingContributors

**Comment by hernessteve** [comment]
I don't eat sushi anymore. Last time i had a food poisoning and felt very and very bad. Very glad i had some Imodium which i ordered from [online pharmacy](https://www.canadadrugsdirect.com/online-pharmacy) some time ago.  One rough night and next morning everything was almost normal. Still if you want to have good quality sushi - you better do it yourself.
